internal/cmd/testdata/github/generator_config.yml

```diff
@@ -23,6 +23,17 @@ resources:
           repo: name
 
 data_sources:
+  gists:
+    read:
+      path: /gists/{gist_id}
+      method: GET
+    schema:
+      ignores:
+        - fork_of.forks
+        - fork_of.history
+        - fork_of.files.language
+        - forks.user.plan.collaborators
```

Comment:
Gists is a good example of why ignores can be useful, as it's not possible to map this data source without using ignores 😃

Reply:
I'm sold on the usefulness of ignores 😄. Should `fork_ok.forks` read `fork_of.forks`?

Reply:
Yes! Did not catch that typo lolol. Updated my comment 👍🏻